index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/136dc91701.js" crossorigin="anonymous"></script>
+    <title>CC Global summit 2015</title>
+</head>
+<body>
+    <header>
+        <div>
+            <nav>
+             <ul class="headerTop">
+                <li class="fb"><a href=""><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li class="twit"><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="eng"><a href="">English</a></li>
+                <li class="pg"><a href="">My Page</a></li>
+                <li class="ld"><a href="">Logout</a></li>
+             </ul>
+            </nav>
+        </div>
+        <div class="headerBtm">
+            <div class="logoSite">
+                <img class="logo" src="./icon/gentle.jpeg" alt="Logo" onclick="fun()">
+                <b>Ideal <br> Men</b>
+            </div>
+            <nav>
+            <ul  class="mainNavBar">
+                 <li class="about"><a href="">About</a></li>
+                 <li><a href="">Program</a></li>
+                 <li><a href="">Join</a></li>
```

Comment:
- [ ] Please make sure that the about page is linked to the proper page and that the about page follows the design for desktop devices.